app/controllers/teams_controller.rb

```diff
@@ -31,6 +31,24 @@ def create
     end
   end
 
+  # PATCH/PUT /teams/1
+  # PATCH/PUT /teams/1.json
+  def update
+    authorize @team
+    team_params = params.require(:team).permit(:description)
+    old_description = @team.description
+    @team.update(team_params)
+
+    @team.create_activity :change_team_description,
+                          owner: current_user,
+                          recipient: @team,
+                          parameters: {
+                            old_description: old_description,
+                            new_description: team_params['description']}
+
+    render js: "window.location = '/teams/#{@team.id}'"
```

Comment:
No it's not really needed. You are right. I will move the reload to app/views/teams/update.js.erb and limit the reloading to the specific div-tag.